README.md

```diff
@@ -129,39 +129,26 @@ Ask for help on the IRC channel: #cordova on irc.freenode.net.
 Writing a Blog Post
 ===================
 
-Use the grunt scripts!
-
-    # Clone apache-blog-posts repo if you haven't already
-    $ grunt cloneBlog
-
-    # Create your blog posts in the apache-blog-posts (or on github) directory, and share with other for review via `git commit`. If creating a new md file, use an earlier one as a template.
-
-    # pull down the latest shared edits of apache-blog-posts
+1. Pull down the latest website codebase for the current posts
     git pull
```

Comment:
Markdown needs an empty line before code, otherwise it bunches it into the list item.